packages/terra-dropdown-button/src/_DropdownList.module.scss

```diff
@@ -17,4 +17,9 @@
     padding-right: 0;
     padding-top: var(--terra-dropdown-button-list-padding-top, 0.42857rem);
   }
+
+  .dropdown-list li:focus {
+    outline: 2px dashed var(--terra-dropdown-button-list-keyboard-focus-outline, #000);
```

Comment:
let's not add new theme variable here. we can make use of the existing one `--terra-dropdown-button-menu-item-focus-outline` and `--terra-dropdown-button-menu-item-focus-outline-offset`

Reply:
Implemented.